examples/popup.js

```diff
@@ -29,9 +29,12 @@ closer.onclick = function() {
 /**
  * Create an overlay to anchor the popup to the map.
  */
-var overlay = new ol.Overlay({
-  element: container
-});
+var overlay = new ol.Overlay(/** @type {olx.OverlayOptions} */ ({
+  element: container,
+  autoPanAnimation: {
+    'duration': 250
```

Comment:
Using quotes around `duration` makes it sound like quotes are mandatory, which should not be the case.